src/sb_gazebo/scripts/teleop_twist_keyboard.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+import roslib; roslib.load_manifest('teleop_twist_keyboard')
```

Comment:
Don't have to do `roslib.load_manifest` since we're building with catkin (I could not launch the node with that line in the script).
This is mentioned in section 2 of the other [style guide](http://wiki.ros.org/PyStyleGuide) I've posted in another comment.